tachyon/crypto/hashes/sponge/poseidon2/poseidon2_unittest.cc

```diff
@@ -53,7 +53,7 @@ TEST_F(Poseidon2GoldilocksTest, Copyable) {
 
   auto config = Poseidon2Config<Params>::Create(
       crypto::GetPoseidon2InternalDiagonalArray<Params>());
```

Comment:
This change doesn't necessarily appear to apply to this commit, though I understand why it's here. See my previous comment for a fix(?)